jenkins_pipelines/environments/uyuni-prs-ci-tests

```diff
@@ -35,6 +35,14 @@ node('pull-request-test') {
         first_env = 1;
         last_env = 10;
         email_to = params.email_to;
+        if (platform_localisation == "NUE") {
+          domain = "mgr.suse.de"
```

Comment:
Personally, and really just a matter of taste, I would have both domains as constants in `pipeline-pull-request-envs.groovy`. 
Then use the `platform_localisation` passed as string parameter in the pipeline to calculate which domain to use inside `pipeline-pull-request-envs.groovy`.

The reasoning about it, is that those domains might change over the years and we might have multiple pipelines using the `pipeline-pull-request-envs.groovy`, so it looks more maintainable fmpov.